src/illuminatio/illuminatio.py

```diff
@@ -94,6 +119,7 @@ def cli(incluster, kubeconfig):
     help="CRI socket used for the interaction with the container runtime",
 )
 def run(
+    test_cases: str,
     outfile: str,
     brief: bool,
     dry: bool,
```

Comment:
There already is this dry-run option that (according to the help text) does what you implemented. I think your implementation is better UX, but this --dry option should be removed.

Reply:
Not exactly - if specified, `--dry-run` avoids running the tests entirely.
`--test-cases` allows you to specify your own test cases, which skips the test generation step, but still executes the tests.
If omitted, the regular test generation happens and the tests are executed.

Reply:
sorry, I wasn't entirely clear: My point is: when generating test cases can be done in a separate command (added by you in this MR), and `--dry-run` does the same as your new command (just worse), then `dry-run` should be removed entirely.

Reply:
Now I get it. Will remove the option and push in a sec.